### ISSUE TITLE
Guest mode + payment request api lead to chrome crash (desktop)

### ISSUE THREAD

**Issue by it-ony** [it-ony/payment-request#11]
Chrome 60 on a Mac or Canary crash, whenever a payment request with event listeners for address change and shipping option change is invoked and the signed in user is a guest user.

```
Process:         Google Chrome Canary [22688]
Path:            /Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary
Identifier:      com.google.Chrome.canary
Version:         62.0.3170.0 (3170.0)
Code Type:       X86-64 (Native)
Parent Process:  launchd [189]
Responsible:     Google Chrome Canary [22688]
User ID:         501

Date/Time:       2017-07-30 00:55:18.446 +0200
OS Version:      Mac OS X 10.9.5 (13F1911)
Report Version:  11
Anonymous UUID:  45045AEA-0830-49AB-F93A-A8C65174AC23

Sleep/Wake UUID: A95F0A93-C2BE-4900-AEAE-2B22F70871D8

Crashed Thread:  0  CrBrowserMain  Dispatch queue: com.apple.main-thread

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Codes: KERN_INVALID_ADDRESS at 0x0000000000000028

VM Regions Near 0x28:
--> 
    __TEXT                 000000010d643000-000000010d644000 [    4K] r-x/rwx SM=COW  /Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary
```

**Comment by rsolomakhin** [comment]
Can you open chrome://crashes and send us the crash ID number? That way we can lookup the stack in our system and fix this quicker.

**Comment by rsolomakhin** [comment]
cc @anthonyvd

**Comment by it-ony** [comment]
*093d7ccd88000000*.

But seems to be fixed in Canary 62.0.3186.0.
